growi/app/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG version=6
+FROM weseek/growi:${version}
+LABEL maintainer Yuki Takei <yuki@weseek.co.jp>
+
+# install dockerize
+ENV DOCKERIZE_VERSION v0.6.1
+USER root
+RUN apt-get update && apt-get install -y curl \
```

Comment:
* [hadolint] warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`[DL3008](https://github.com/hadolint/hadolint/wiki/DL3008)
* [hadolint] warning: Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check[DL4006](https://github.com/hadolint/hadolint/wiki/DL4006)
* [hadolint] info: Avoid additional packages by specifying `--no-install-recommends`[DL3015](https://github.com/hadolint/hadolint/wiki/DL3015)